.github/actions/regression-tests/action.yml

```diff
@@ -87,10 +87,6 @@ runs:
         TIMEDOUT_STEP_1: ${{ steps.first.outputs.TIMEDOUT }}
         TIMEDOUT_STEP_2: ${{ steps.second.outputs.TIMEDOUT }}
       run: |
-        echo "🪵🪵🪵🪵🪵🪵 Latest log before timeout 🪵🪵🪵🪵🪵🪵\n\n"
```

Comment:
I removed this. Although it can be helpful in certain cases it's also annoying because a lot of the tests do not write to a single log, so printing only one them is not that useful and is also noisy. Since now we upload only the failed logs, it should be straightforward to download + open them. 

If you have strong objections against this I am happy to revert.

Reply:
I do not understand, sorry.
The change in this PR is that you dont print the failed test logs but just upload them?

In here is see you removed a section of last log before timeout. So do we upload logs for timeout tests?

Reply:
> The change in this PR is that you dont print the failed test logs but just upload them? 
Nope, I removed printing the last log of the last failed test (since the last test can contain many logs) because it was not really useful most of the time. 

In addition, for each test run, we used to print the corresponding logs. This was annoying because we are only interested in the logs of the tests that failed. So I removed that as well and now we only print the name of the logs of the failed tests. See here for an example: https://github.com/dragonflydb/dragonfly/actions/runs/7798539464/job/21267436332#step:6:178 (other tests do not print this). 

Moreover, previously we also uploaded all the logs (both from the failed and successful tests). This was again annoying so we now only upload the logs of the failed tests.